python-sdk/nuscenes/eval/panoptic/panoptic_track_evaluator.py

```diff
@@ -160,6 +196,8 @@ def add_batch_panoptic(self,
             self.gts[scene] = [{} for _ in range(self.n_classes)]
             self.intersects[scene] = [{} for _ in range(self.n_classes)]
             self.intersects_ovr[scene] = [{} for _ in range(self.n_classes)]
+            self.instance_preds[scene] = {}
+            self.instance_gts[scene] = [{} for _ in range(self.n_classes)]
         # Make sure instance IDs are non-zeros. Otherwise, they will be ignored. Note in nuScenes-panoptic,
```

Comment:
`nuScenes-panoptic` --> `Panoptic nuScenes`?

Reply:
Corrected it